app/views/users/index.html.erb

```diff
@@ -1,7 +1,17 @@
-<!DOCTYPE HTML>
+<!DOCTYPE html>
 <html>
   <head></head>
   <body>
-    <h1>SHOW USERS</h1>
+    <div class='container'>
+      <% @users.each do |user| %>
```

Comment:
- [Optional] Hi @HanaSabih , instead of using @objects.each(`users.each`) a better practice is to utilize Rails partial. Ex. you could have created a  user partial in #app/views/users/_user.html.erb and display the user name, photo, and posts_counter then render the user partial in app/views/users/index.html.erb as below:
`<%= render @users %>`
This way, your codebase is DRY since you'll only be required to render in all views instead of repeating the same code as in the posts index and show page.